lib/wallaby/browser.ex

```diff
@@ -822,8 +822,8 @@ defmodule Wallaby.Browser do
   @doc """
   Matches the Element's content with the provided text and raises if not found
   """
-  @spec assert_text(Element.t(), String.t()) :: boolean()
-  @spec assert_text(parent, Query.t(), String.t()) :: boolean()
+  @spec assert_text(Element.t(), String.t()) :: parent
+  @spec assert_text(parent, Query.t(), String.t()) :: parent
```

Comment:
The docs and specs should be updated to reflect the actual types that are permitted. (I know this isn't related to your change).

`assert_text` and `has_text?` can both take sessions or elements, but the specs and docs don't reflect that.

Can you change the specs and docs to be accurate, as well as well as enhance the docs with an example? It would be very much appreciated.